lang/cs/Org.Apache.REEF.Tests/Functional/RuntimeName/RuntimeNameTest.cs

```diff
@@ -33,27 +37,76 @@ namespace Org.Apache.REEF.Tests.Functional.Driver
     public class RuntimeNameTest : ReefFunctionalTest
```

Comment:
@markusweimer I am not sure how, as I need the runtime to be initialized to test the java-->c# pass. It looks like the easiest way that would give the best coverage.